distrobuilder/main_incus.go

```diff
@@ -352,6 +353,13 @@ func (c *cmdIncus) run(cmd *cobra.Command, args []string, overlayDir string) err
 			return fmt.Errorf("Failed to mount UEFI partition: %w", err)
 		}
 
+		rootfsDevUUID, err := vm.findRootfsDevUUID()
+		if err != nil {
+			return fmt.Errorf("Failed to find rootfs device UUID: %w", err)
+		}
+
+		c.global.ctx = context.WithValue(c.global.ctx, shared.ContextKeyEnviron,
+			[]string{fmt.Sprintf("ROOTFS_DEVICE_UUID=%s", rootfsDevUUID)})
```

Comment:
I'd probably go for `DISTROBUILDR_ROOTFS_UUID` instead so that it's clear that this is a distrobuilder thing and we avoid potential conflicts with other env variables (as unlikely as that may be).

Reply:
done